project1/application.py

```diff
@@ -116,3 +119,26 @@ def userHome(Username):
     if Username in session:
         return render_template("userDetails.html", username=Username, message="Successfully logged in.", heading="Welcome back")
     return redirect(url_for('index'))
+
+
+
+
+
+
+
+
+
+@app.route("/bookpage/<isbn>",methods=["GET","POST"])
+def bookpage(isbn):
+	
+	bk = isbn
+
+	book = Books.query.filter_by(isbn=bk).first()
+	
+	if request.method == "POST":
```

Comment:
why is a POST request required for bookpage?

Reply:
It is required to bring the review details to the book page.